utils_cv/detection/dataset.py

```diff
@@ -47,21 +68,21 @@ def __init__(
             batch_size: batch size for dataloaders
             transforms: the transformations to apply
             train_test_ratio: the ratio of training to testing data
-            image_folder: the name of the image folder
+            im_folder: the name of the image folder
             annotation_folder: the name of the annotation folder
         """
 
         self.root = Path(root)
         self.transforms = transforms
-        self.image_folder = image_folder
-        self.annotation_folder = annotation_folder
+        self.im_folder = im_folder
```

Comment:
Do we use "folder" or "dir" for classification? I found one example for "dir" for now, but not sure that applies generally to our utils. Ideally this here should be following the example what is already in utils_cv.